build.sbt

```diff
@@ -243,9 +242,11 @@ lazy val V = new {
   def scala2Versions = nonDeprecatedScala2Versions ++ deprecatedScala2Versions
 
   // Scala 3
-  def scala3RCVersions = Seq("3.0.0-RC3", "3.0.0-RC2", "3.0.0-RC1")
   def nonDeprecatedScala3Versions =
-    scala3 +: scala3RCVersions :+ nextScala3RC
+    Seq(
```

Comment:
Can it be oneline? Just change it if you do any further fixes, otherwise I would leave it.